server.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const package_info = require('./package.json');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 8080;
+const server_name = package_info.serverName;
+const server_version = package_info.serverVersion;
+
+const json_response = {
```

Comment:
Can't we migrate to api method?

Reply:
What do you mean by api method and api comments?